test_chameleon.py

```diff
@@ -13,14 +13,14 @@ def test_basic_json_service(self):
         responds to queries as expected.
         Port defaults to 8080.
         """
-        settings_list = [{"url": "/path1", 
-                          "status_code": 200, 
-                          "content_type": "application/json", 
+        settings_list = [{"url": "/path1",
+                          "status_code": 200,
+                          "content_type": "application/json",
                           "content": '{"foo": "bar"}'}]
         chameleon.start(settings_list)
         response = requests.get('http://localhost:8080/path1')
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.content, '{"foo": "bar"}')
+        self.assertEqual(response.content, b'{"foo": "bar"}')
```

Comment:
Yes, Py2.7 ignores b.
In Py3 there's a distinct difference between bytes and chars.

The build passes for various versions - refer the Travis yaml file :neckbeard: